demos/interactive_face_detection_demo/interactive_face_detection.hpp

```diff
@@ -55,80 +48,18 @@ static const char target_device_message_lm[] = "Optional. Target device for Faci
-
-/// @brief Message for the maximum number of simultaneously processed faces for Facial Landmarks Estimation network
-static const char num_batch_lm_message[] = "Optional. Number of maximum simultaneously processed faces for Facial Landmarks Estimation network " \
-"(by default, it is 16)";
-
-/// @brief Message for dynamic batching support for AgeGender net
-static const char dyn_batch_ag_message[] = "Optional. Enable dynamic batch size for Age/Gender Recognition network";
-
-/// @brief Message for dynamic batching support for HeadPose net
-static const char dyn_batch_hp_message[] = "Optional. Enable dynamic batch size for Head Pose Estimation network";
-
-/// @brief Message for dynamic batching support for Emotions net
-static const char dyn_batch_em_message[] = "Optional. Enable dynamic batch size for Emotions Recognition network";
-
-/// @brief Message for dynamic batching support for Facial Landmarks Estimation network
-static const char dyn_batch_lm_message[] = "Optional. Enable dynamic batch size for Facial Landmarks Estimation network";
-
-/// @brief Message for performance counters
-static const char performance_counter_message[] = "Optional. Enable per-layer performance report";
-
-/// @brief Message for GPU custom kernels description
-static const char custom_cldnn_message[] = "Required for GPU custom kernels. "\
-"Absolute path to an .xml file with the kernels description.";
-
-/// @brief Message for user library argument
-static const char custom_cpu_library_message[] = "Required for CPU custom layers. " \
-"Absolute path to a shared library with the kernels implementation.";
-
 /// @brief Message for probability threshold argument
 static const char thresh_output_message[] = "Optional. Probability threshold for detections";
 
-/// @brief Message for face enlarge coefficient argument
-static const char bb_enlarge_coef_output_message[] = "Optional. Coefficient to enlarge/reduce the size of the bounding box around the detected face";
-
-/// @brief Message raw output flag
-static const char raw_output_message[] = "Optional. Output inference results as raw values";
-
-/// @brief Message do not wait for keypress after input stream completed
-static const char no_wait_for_keypress_message[] = "Optional. Do not wait for key press in the end.";
-
 /// @brief Message do not show processed video
 static const char no_show_processed_video[] = "Optional. Do not show processed video.";
 
-/// @brief Message for asynchronous mode
-static const char async_message[] = "Optional. Enable asynchronous mode";
-
-/// @brief Message for shifting coefficient by dx for detected faces
-static const char dx_coef_output_message[] = "Optional. Coefficient to shift the bounding box around the detected face along the Ox axis";
-
-/// @brief Message for shifting coefficient by dy for detected faces
-static const char dy_coef_output_message[] = "Optional. Coefficient to shift the bounding box around the detected face along the Oy axis";
-
```

Comment:
Please keep these options. Those can be parameters to `ParseSSD`

Reply:
Done